app/src/common/AppHeader.module.scss

```diff
@@ -3,6 +3,8 @@
         display: flex;
         padding-left: 18px;
         padding-right: 18px;
+        align-items: center;
+        min-height: 60px;
```

Comment:
Before/ after this changes

<img width="166" alt="image" src="https://github.com/epam/UUI/assets/11524637/d846b552-b5b8-41a3-b5bd-b08e437d7704"> 
<img width="166" alt="image" src="https://github.com/epam/UUI/assets/11524637/0fcbe6a5-bcac-41f5-8939-ffad28f0c33c">